packages/core/src/integrations/extraerrordata.ts

```diff
@@ -115,7 +115,12 @@ function _extractErrorData(
     // Error.cause is a standard property that is non enumerable, we therefore need to access it separately.
     // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause
     if (captureErrorCause && error.cause !== undefined) {
-      extraErrorInfo.cause = isError(error.cause) ? error.cause.toString() : error.cause;
+      if (isError(error.cause)) {
+        const causeName = error.cause.name || error.cause.constructor.name;
```

Comment:
Maybe change the variable name here to make it easier to understand that this is not the name of the cause but the error name as this is of type `Error`.

Or `causeErrorName`.

```suggestion
        const errorName = error.cause.name || error.cause.constructor.name;
```

Reply:
Thank you for prompt review, good suggestion 👍